src/datagen/generated/domum_ornamentum/data/domum_ornamentum/tags/blocks/pillar_materials.json

```diff
@@ -0,0 +1,14 @@
+{
+  "replace": false,
+  "values": [
+    "minecraft:clay",
```

Comment:
I hadn't added clay because it didn't seem for me like a "stable material"

Reply:
removed some of the less 'stable' materials.